### DISCUSSION TITLE
Hardware audio controll

### DISCUSSION THREAD

**Discussion by flibble666** [Wilsondotzip/VolumeMaster#3] [Ideas]
this project maybe useful for info on hardware controls https://maxmixproject.com/build/.

**Comment by Wilsondotzip** [comment]
Update on this: 
VoiceMeeter integration is currently being worked on, this means that you will be able to use HAM to control hardware devices through VoiceMeeter. Not as elegant as direct control, but works well with the benefit of also having general VoiceMeeter integration for other purposes.
